roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -0,0 +1,25 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerized
+
+
+class PackageVersion(NotContainerized, OpenShiftCheck):
+    """Check that available RPM packages match the required versions."""
+
+    name = "package_version"
+    tags = ["preflight"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return (
+            super(PackageVersion, cls).is_active(task_vars)
+            and task_vars.get("deployment_type") == "openshift-enterprise"
```

Comment:
This particular check looking up `deployment_type` is a bug: https://trello.com/c/NdWoIydb
This check applies equally well to any deployment_type, the package names should not be hard-coded in the `aos_version.py` module...

I would fix it in a separate PR, but if it helps reduce the confusion I'll do it in a new commit.